lib/constants.js

```diff
@@ -19,6 +19,8 @@ exports.BOOTSTRAP_NODES = [
   '138.68.147.8@node3.hyperdht.org:49737'
 ]
 
+exports.KNOWN_NODES = global?.Pear?.config?.dht || []
```

Comment:
doing global.Pear?.config.dht should be fine, global is always set and config is always set if Pear is